CONTRIBUTING.md

```diff
@@ -76,6 +76,8 @@ using the command `make checkstatic` and fix any resulting errors. Otherwise,
 would affect the data or custom code of a study - document this in your pull 
 request description and tag it with **Caveat for Existing Projects**. 
 This helps us to document our release notes.
+* In addition to the above guidelines, we use [Alex JS](https://github.com/get-alex/alex) 
```

Comment:
This still shouldn't be advertising external tools. The HelpStyleGuide mention of it is fine, but for CONTRIBUTING.md the contributor should just need to know that they can run `make checklanguage` to verify the documentation.

Reply:
Done